wurst/systems/Knockback3.wurst

```diff
@@ -0,0 +1,225 @@
+/** By default, this package allows units to collide with destructables and
+    destroy them. To disable this feature, make the global
+    destroyDestructableSpeedThreshold a very high value:
+
+       Knockback3.destroyDestructableSpeedThreshold = 999999999.
```

Comment:
The value is `var` not `constant`, so my example is meant to show how to change it from the context of some function that imports Knockback3, not suggesting modifying it.

Do you still prefer constant with `@configurable`? If so, is that just convention?